### ISSUE TITLE
Accessibility - Poor Color Contrast

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor#65]
The light gray text on a bunch of the screens can be an issue for those with color contrast issues with their vision because of poor color contrast.

See example screens and information about them below.

**Statistics Screen**

The industries under “Working” such as “Agriculture”, “Services”, “Industry”, etc. and statistics of “Unavailable” have poor color contrast.

**Enforcement and Legal Standards Screen**

The “Unknown” and “N/A” text, along with the asterisks have poor color contrast for the above screens.

There is poor color contrast for the "footnotes" in the screen footer.

A good example of all these issues can be found on the country of Kiribati screen.

My suggestion to fix the problem by making the text the same color as the other black text.

**Comment by aquilent-dgemoets** [comment]
For what it's worth, these colors are unchanged from last year.

But at ILAB request we could send out a before/after mockup in which the indicated grey text is changed to black.  It's possible the screen would lose some of its "name/value" look-and-feel, but color contrast is important, too.

**Comment by jfc3-dol** [comment]
If you still want that text to be lighter you could use a darker gray but not black such as #6E6E6E or #666666 which will give you a color contrast ratio of 5.1:1 when you need 4.5:1 to pass WCAG 2.0 AA.

**Comment by aquilent-dgemoets** [comment]
This is addressed in the latest beta (9/9/16).  Please re-test and close the issue if it is satisfactory.

**Comment by jfc3-dol** [comment]
The light gray text on the countries statistics screen in the "Working" section has a color contrast ratio of 3.03:1 and should be 4.5:1. 
![ilab ios countries statics poor color contrast](https://cloud.githubusercontent.com/assets/10423487/18682518/9d8d13d4-7f3a-11e6-92c4-d152590a9b7d.PNG)

**Comment by aquilent-dgemoets** [comment]
That's strange.  As you can see in the screenshot, the developer set the color to #6E6E6E.  I'm not sure why it's not meeting your specifications.  Maybe iOS is doing something after the fact?  We're not sure what to do other than specify the requested color.  I guess we can try something darker and see if it is compliant?  This will have to happen in a post-9/30 release.
![issue65](https://cloud.githubusercontent.com/assets/15035422/18723129/6dbffb14-8005-11e6-83a0-44357e5f5cab.png)

**Comment by jfc3-dol** [comment]
Yes, #6e6e6e has enough color contrast.

Are they using some transparency or some kind of transform, etc. because I found that caused issues with the color contrast on other web based projects I have done accessibility reviews on before?

**Comment by aquilent-dgemoets** [comment]
It must be doing something, since I looked up the color on http://web.stanford.edu/class/cs101/image-rgb-explorer.html and it's definitely darker than what we have on iOS.  We'll try a darker color and hope the ratio is ok if there's time after 9/30.

**Comment by jfc3-dol** [comment]
Okay. 

#3d3d3d works well since it's more of a dark charcoal gray but not pure black.

We used it over at ARMY.mil so we got enough contrast but didn't go to high as using #000000.

**Comment by carolynhuang** [comment]
Please confirm whether this is still an open issue. From previous discussion, it appears to be resolved.

**Comment by jfc3-dol** [comment]
Issues seems to be corrected now.

Closing issue.
